modules/ROOT/pages/platform/user-management.adoc

```diff
@@ -25,23 +25,36 @@ You can access the **Team Members** page by selecting **Team Members** from the
 
 === Roles
 
-Users within a team can have one of two roles, _Admin_ or _Member_.
+Users within a team can be assigned one of the following roles:
```

Comment:
If you want to use a table, you can use something like this:

```
:check-mark: icon:check[]

.Roles
[opts="header",cols="3,1,1,1"]
|===
| Capability | Admin | Member | Viewer
| View the team member and their roles | {check-mark} | {check-mark} | {check-mark}
| View and open instances | {check-mark} | {check-mark} | {check-mark}
| Perform all actions on instancesfootnote:[Actions include creating, deleting, pausing, resuming and editing instances.] | {check-mark} | {check-mark} |
| Clone data to new and existing instances | {check-mark} | {check-mark} |
| Take on-demand snapshots | {check-mark} | {check-mark} |
| Restore from snapshots | {check-mark} | {check-mark} |
| Invite new users to the team | {check-mark} | |
| Edit existing team members' roles | {check-mark} | |
| Delete existing team members from the team | {check-mark} | |
| View and edit billing information | {check-mark} | |
|===
```

The footnote should work fine.

(The `check-mark` attribute is not necessary, it just makes it easier to change the actual icon if needed.)